tensornetwork/ncon_interface.py

```diff
@@ -35,45 +36,48 @@ def ncon(tensors: Sequence[Union[network_components.BaseNode, Tensor]],
 
     If a contraction order `con_order` and an output order `out_order`
     are both provided, the edge labels can be anything.
-    Otherwise (`con_order == None or out_order == None`), the edge labels 
+    Otherwise (``con_order == None or out_order == None``), the edge labels 
```

Comment:
Can you be sure to only use one grave accent (`) ? Using 2 changes the color of the display to red and it looks kinda bad.

See this for example. The red color are the ones with 2 and the black ones use just 1. 
![image](https://user-images.githubusercontent.com/8702042/72850174-5e4dbf00-3c5d-11ea-8553-9aa69aaaa961.png)

Reply:
Done.  ^^

A little follow-up of my motivation before:
I was basically trying to mimic JAX and it's awesome docs, and it seemed to me that their approach is this:
Single accent for referencing or inter-linking something, or just to bring attention
Double accent for small code examples or variable names/data types

Reply:
JAX does have very nice documentation. We should try to mimic them whenever possible.